### DISCUSSION TITLE
Feature Request: Reducing Balance Depreciation Method

### DISCUSSION THREAD

**Discussion by aungzwin10** [ledgersmb/LedgerSMB#7090] [Ideas]
Is it possible to add another depreciation method in the Fixed Asset Module?
The requested method is `Reducing Balance Method`

**Comment by ehuelsmann** [comment]
It's possible: you need an extra depreciation method like the one in https://github.com/ledgersmb/LedgerSMB/blob/master/sql/modules/Assets.sql#L141

**Comment by ehuelsmann** [comment]
Converting to an idea as this is not a priority for the project right now.

**Reply by ehuelsmann** [reply]
I'll build it when a sponsor comes along.
